hdf5.spec

```diff
@@ -587,6 +583,8 @@ done
 %changelog
 * Wed Jul 22 2020 Maureen Jean <maureen.jean@intel.com> - 1.12.0-1
 - Update HDF5 to version 1.12.0
+- Obsolete the previous version of hdf5 package with this build
+- No longer packaging example files
```

Comment:
Was this not resolved so that we didn't need to regress this version of the package?  Was it investigated and determined to be too difficult to fix?

Reply:
Can you take a look?  I can not determine why the examples are trying to build in /usr/     The Makefile.am is located in BUILD/examples/Makefile.am

Reply:
I can tomorrow.  My day is running out and I want to get a fix for the hdf5 master failure landed before I go which is in #10 if you could review that please.